plugins/kuttl.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kuttl
+spec:
+  version: "v0.1.0"
+
+  shortDescription: Declaratively build, install, and run operators using KUTTL.
+  homepage: https://kudo.dev/
+  description: |
+    The KUbernetes Test TooL (KUTTL) is a highly productive test
+    toolkit for testing operators on Kubernetes.
+  caveats: |
+    See the documentation for more information: https://kudo.dev/docs/
+
+  platforms:
+  - selector:
+      matchLabels:
+        os: "linux"
+        arch: "amd64"
+    uri: https://github.com/kudobuilder/kuttl/releases/download/v0.1.0/kuttl_0.1.0_linux_x86_64.tar.gz
+    sha256: "4043a3052a0f5b6c66b80fb1ad8dcfba10cb1e3a7f40812f991e1dd304a5af90"
+    bin: "./kubectl-kuttl"
+    files:
```

Comment:
This is the default. 
You can omit files: in this case. 
Do your tarballs have a LICENSE?

Reply:
no license in the tarballs

Reply:
When I checked, there was a license 😉